<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnalyzingPopPage</class>
 <widget class="QWidget" name="AnalyzingPopPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1142</width>
    <height>826</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_4" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>39</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color: rgba(200, 200, 200, 0.47)</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>160</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>600</width>
       <height>500</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#widget{
background-color: rgba(200, 200, 200, 0.47);
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>218</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="Box" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>700</width>
          <height>500</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#Box{
border-radius:20px;
border-image: url(:/AnalyxingPopPage/images/AnalyzingPage/Box.svg);
}</string>
        </property>
        <widget class="QPushButton" name="closeBtn">
         <property name="geometry">
          <rect>
           <x>640</x>
           <y>10</y>
           <width>51</width>
           <height>51</height>
          </rect>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/AnalyxingPopPage/images/AnalyzingPage/close.svg</normaloff>:/AnalyxingPopPage/images/AnalyzingPage/close.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>240</x>
           <y>310</y>
           <width>221</width>
           <height>51</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color:rgb(111, 111, 111);
font-size:42px;</string>
         </property>
         <property name="text">
          <string>代码分析中</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QWidget" name="widget_3" native="true">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>30</y>
           <width>311</width>
           <height>271</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/AnalyxingPopPage/images/AnalyzingPage/analyzingImage.png);</string>
         </property>
        </widget>
        <widget class="QLabel" name="percentText">
         <property name="geometry">
          <rect>
           <x>570</x>
           <y>395</y>
           <width>101</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size:24px;</string>
         </property>
         <property name="text">
          <string>?/?</string>
         </property>
        </widget>
        <widget class="QFrame" name="PercentageBarBg">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>400</y>
           <width>491</width>
           <height>20</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">#PercentageBarBg {
border: 1px solid #FCBB83 ;
border-radius: 5px;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QFrame" name="PercentageBar">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>0</width>
            <height>20</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">#PercentageBar{
background-color: #FCBB83;
border: 1px solid #FCBB83;
border-radius: 5px;

}</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="popDlg" native="true">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>130</y>
           <width>511</width>
           <height>211</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">#popDlg{
	border-image: url(:/AnalyxingPopPage/images/AnalyzingPage/innerPopPage.svg);
border-radius:20px;
}</string>
         </property>
         <widget class="QPushButton" name="popPage_closeBtn">
          <property name="geometry">
           <rect>
            <x>450</x>
            <y>20</y>
            <width>41</width>
            <height>29</height>
           </rect>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="images.qrc">
            <normaloff>:/AnalyxingPopPage/images/AnalyzingPage/close.svg</normaloff>:/AnalyxingPopPage/images/AnalyzingPage/close.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>60</y>
            <width>441</width>
            <height>71</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color:rgb(111, 111, 111);
font-size:32px;</string>
          </property>
          <property name="text">
           <string>正在分析项目，是否取消分析？</string>
          </property>
         </widget>
         <widget class="QPushButton" name="popPage_cancleBtn">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>150</y>
            <width>121</width>
            <height>31</height>
           </rect>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">border-image: url(:/AnalyxingPopPage/images/AnalyzingPage/innerBtnY.svg);
color:rgb(111, 111, 111);
font-size:22px;</string>
          </property>
          <property name="text">
           <string>确认取消</string>
          </property>
         </widget>
         <widget class="QPushButton" name="popPage_closeBtn2">
          <property name="geometry">
           <rect>
            <x>300</x>
            <y>150</y>
            <width>121</width>
            <height>31</height>
           </rect>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">border-image: url(:/AnalyxingPopPage/images/AnalyzingPage/innerBtnN.svg);
color:rgb(111, 111, 111);
font-size:22px;</string>
          </property>
          <property name="text">
           <string>再想想</string>
          </property>
         </widget>
        </widget>
        <widget class="QLabel" name="AnalyzeStepText">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>435</y>
           <width>561</width>
           <height>51</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color:rgb(111, 111, 111);
font-size:22px;</string>
         </property>
         <property name="text">
          <string>分析中</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>218</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_5" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>28</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color: rgba(200, 200, 200, 0.47)</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>160</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
